### DISCUSSION TITLE
Auto.proxy supports X-WEBAUTH-ORG header

### DISCUSSION THREAD

**Discussion by kaivean** [grafana/grafana#47385] [Plugins]
## Why

now Auto.proxy  supports X-WEBAUTH-USER、X-WEBAUTH-NAME、X-WEBAUTH-ROLE.

eg:
```
setHeader('X-WEBAUTH-USER', '');
setHeader('X-WEBAUTH-NAME', '');
```
But users are registerd to the org `1` or   `auto_assign_org_id` from the ini. 

## What
I expect to set custom org id by adding X-WEBAUTH-ORG header.  Because the different products has different org in my business system. So need to sync users of specific product to specific org 

eg: 
```
setHeader('X-WEBAUTH-ORG', '2');
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by enchi1o** [reply]
I have the same business requirement. I hope to set the user’s organization in the auth proxy and allow this user to operate within the specified organization.
